rllib/algorithms/ppo/ppo.py

```diff
@@ -325,8 +326,9 @@ def validate_config(self, config: TrainerConfigDict) -> None:
             config["train_batch_size"] > 0
             and config["train_batch_size"] % calculated_min_rollout_size != 0
         ):
-            new_rollout_fragment_length = config["train_batch_size"] // (
-                num_workers * config["num_envs_per_worker"]
+            new_rollout_fragment_length = math.ceil(
```

Comment:
Great idea! Yeah, this should push the collected num samples only slightly over the `train_batch_size` so a new rollout is not required, which would potentially almost double the train batch.